src/wrappers/themis/ruby/rbthemis.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name        = 'rbthemis'
-  s.version     = '0.13.0'
-  s.date        = '2020-07-08'
+  s.version     = '0.14.0'
+  s.date        = '2021-12-24'
```

Comment:
Do we actually plan to release exactly on Christmas day? :)

Reply:
> Do we actually plan to release exactly on Christmas day? :)

I had to put *something* there vOv

The closest other “nice” date is the solstice, 2021-12-21. Are we going to make it? :D

Reply:
we def won't make it 2021-12-21